Gemfile

```
@@ -10,15 +10,14 @@ gem 'rails-controller-testing'
-                { 'github': 'spree/spree', 'branch': '5-3-stable' }
+                { 'github': 'spree/spree', 'branch': 'main', 'glob': 'spree/**/*.gemspec' }
              end
 gem 'spree', spree_opts
 gem 'spree_admin', spree_opts
 
-gem 'spree_storefront'
-gem 'spree_page_builder'
-
-gem 'spree_dev_tools'
+gem 'spree_multi_store', github: 'spree/spree-multi-store'
+gem 'spree_legacy_api_v2'
```

RuboCop (1.85.1)

[convention] Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `spree_legacy_api_v2` should appear before `spree_multi_store`.

(Bundler/OrderedGems)